ADAL/src/ADLogger.m

```diff
@@ -315,7 +315,7 @@ + (void)logToken:(NSString *)token
         [logString appendFormat:@" expires on %@", expiresOn];
     }
     
-    AD_LOG_INFO_PII(correlationId, @"%@", logString);
+    AD_LOG_INFO(correlationId, @"%@", logString);
```

Comment:
logString contains token hash. Token hash is PII.

Reply:
fixed